sdk/ai/azure-ai-projects/README.md

```diff
@@ -43,21 +43,29 @@ TODO
 ## Examples
 
 ### Agents
-The following steps outline the typical sequence for interacting with agents:
-
-  - <a href='#create-project-client'>Create a project client</a>
-  - <a href='#create-agent'>Create an agent</a> with <a href='#create-agent-with-toolset-or-tools-and-tool-resources'>toolset, or tools and tool resources</a> including:
-    - <a href='#create-agent-with-file-upload-in-vector-store-for-file-search'>File Search with file upload indexed by vector stores</a>
-    - <a href='#create-agent-with-file-upload-for-code-interpreter'>Code Interpreter with file upload</a>
-    - <a href='#create-agent-with-function-tool'>Function calls</a>
+Agents in the Azure AI Projects client library are designed to facilitate various interactions and operations within your AI projects. They serve as the core components that manage and execute tasks, leveraging different tools and resources to achieve specific goals. The following steps outline the typical sequence for interacting with agents:
```

Comment:
We need to put a comment here saying that Agents is currently in private preview, with contact details on how to ask to be added to the preview. Please follow up with Mads on the proper text here.